MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+include LICENSE
+recursive-include m2cgen VERSION.txt
+recursive-include m2cgen linear_algebra.*
+global-exclude *.py[cod]
```

Comment:
Is it smart enough to exclude all the pycache directories as well?

Reply:
It was taken from the official documentation. I think we can trust them.
https://packaging.python.org/guides/using-manifest-in/ (the very bottom of the page)